Slim/Control/Commands.pm

```diff
@@ -1424,6 +1424,8 @@ sub playlistXitemCommand {
 	if ( $handler && $handler->can('explodePlaylist') ) {
 		$handler->explodePlaylist($client, $path, sub {
 			my $tracks = shift;
+			# transform opml list into url array if needed
+			$tracks = [ map { $_->{play} || $_->{url} } @{$tracks->{items}} ] if ref $tracks eq 'HASH';
```

Comment:
Please improve readability by using an `if {}` block and formatting the map onto multiple lines.

Reply:
oh, you don't like my one-liner habit ;-)

Reply:
Joke aside, I'm happy to do that, but the rest of the file as a few maps like
```
$info[0] = join(', ', map { $_->name } Slim::Schema->search('Genre', { 'id' => { 'in' => [ split(/,/, $genre_id) ] } })->all);
```
(was not me :smile:)

Reply:
Whoever did that didn't have any reviewer trying to understand what he did 😝.

Reply:
yes sir, done :face_with_head_bandage: 

(I scanned through LMS to figure out the use, 90% are one-liner... many former assembly code developers like me)